cmd/operator-ui/actions/blackducks.go

```diff
@@ -156,6 +157,13 @@ func (v BlackducksResource) common(c buffalo.Context, bd *blackduckv1.Blackduck)
 	} else {
 		bd.View.ContainerTags = images
 	}
+
+	kubeConfig, err := rest.InClusterConfig()
```

Comment:
Since synopsys-operator is vendored, you can use the pkg/protoform files to get the kube config

Reply:
I will take care of the UI in a different PR